Directory.Packages.props

```diff
@@ -12,7 +12,7 @@
     <!-- Compatibility -->
     <PackageVersion Include="System.Threading.Channels" Version="$(SystemVersion)" />
     <PackageVersion Include="System.Collections.Immutable" Version="$(SystemVersion)" />
-    <PackageVersion Include="System.Text.Json" Version="$(SystemVersion)" />
+    <PackageVersion Include="System.Text.Json" Version="8.0.4" />
```

Comment:
What is SystemVersion? Why do we need to instead hardcord?

Reply:
The SystemVersion is also hardcoded to 8.0.0 in the file. There was an update 3 days ago to System.Text.Json package which marked that version as high vulnerability, so the build was failing until I changed it to the patched version.